bin/install-if-no-packages.js

```diff
@@ -3,7 +3,7 @@ const fs = require( 'fs' );
 
 if ( ! fs.existsSync( 'node_modules' ) ) {
 	console.log( 'No "node_modules" present, installing dependencies...' );
-	const installResult = spawnSync( 'npm', [ 'ci' ], {
+	const installResult = spawnSync( 'npx', [ 'lerna', 'bootstrap' ], {
```

Comment:
Before, we expected root shrinkwrap to exist, it seems that now we should expect package locks to exist and we could continue to use `CI`.
```suggestion
	const installResult = spawnSync( 'npx', [ 'lerna', 'bootstrap', '--ci' ], {
```